examples/hosts/app-integration/external-data/src/model/taskList.ts

```diff
@@ -34,7 +35,10 @@ class Task extends TypedEventEmitter<ITaskEvents> implements ITask {
     public constructor(
         private readonly _id: string,
         private readonly _name: SharedString,
-        private readonly _priority: ISharedCell<number>
+        private readonly _priority: ISharedCell<number>,
+        public diffName: string = DEFAULT_DIFF_NAME,
```

Comment:
Recommend using accessors rather than making these directly public, same as the _id/_name.  Also consider making them nullable (`string | undefined`) rather than having to use a default for local-only tasks.

Reply:
I tried using (string | undefined) but the react component useEffect threw a real hissy fit about it so I never managed to figure out that pattern. Do you have examples of that that I can copy?

Reply:
What sort of error?  You'll need to typecheck them before using them as string/number but that's the only thing I would expect.

Reply:
I think I was getting tripped up with the typecheck earlier. It used the accesser pattern now and it works great!